src/sqlancer/sqlite3/SQLite3Options.java

```diff
@@ -90,6 +90,13 @@ public class SQLite3Options implements DBMSSpecificOptions<SQLite3OracleFactory>
             "--execute-queries" }, description = "Specifies whether the query in the fuzzer should be executed", arity = 1)
     public boolean executeQuery = true;
 
+    @Parameter(names = {
+            "--max-tables" }, description = "The maximum number of tables/virtual tables/ rtree tables/ views that can be created")
```

Comment:
Could we use `--max-num-tables` here and `--max-num-indexes` below, to match the naming convention of other option classes (see below)?

https://github.com/sqlancer/sqlancer/blob/a535520766808fd5bb71db41a0351e9d1f0d5331/src/sqlancer/duckdb/DuckDBOptions.java#L84

Reply:
Fixed